lektor/admin/static/js/widgets/multiWidgets.jsx

```diff
@@ -55,7 +55,7 @@ const CheckboxesInputWidget = React.createClass({
     let {className, value, placeholder, type, ...otherProps} = this.props  // eslint-disable-line no-unused-vars
     className = (className || '') + ' checkbox'
 
-    const choices = this.props.type.choices.map(function (item) {
```

Comment:
This actually needs to stay using a normal function. Just changing it to an arrow function will result in the `this` inside the function being undefined. See https://github.com/lektor/lektor/pull/427 and the conversation on that pull request.

Reply:
Before I changed this to an arrow function I got errors about `this` being undefined. After changing it to an arrow function, my model worked as intended. Are you certain this particular case needs to be a normal anonymous function?

Reply:
I did some more reading on [arrow functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions) and the important thing here is that they'll not bind `this`, meaning they are free to use the outer scope's `this`. Since both `this.isActive` and `this.onChange` is part of the outer scope this should be an arrow function.

Reply:
It seems I 'fixed' a false positive in #427. This function can be an arrow function (I'll remove the change request). As an arrow function it works fine for me, which you're right should be expected since there is a functional this in the outer scope. Though, I'm not getting an additional 'undefined' in app.js as a normal function (as it was even in the initial commit), and I would expect that to still work too. I'm curious about that. Where are you seeing the error?

Reply:
I haven't isolated it exactly, but it might be related to a list of checkboxes. For this particular case I've made a plugin that reads categories from a databag and displays the contents as checkboxes, but I assume any collection of checkboxes will trigger it.